src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -543,16 +537,14 @@ namespace AirLoopHVACDOAS {
                                                                 _,
                                                                 _,
                                                                 _);
-                        if (errorsFound) {
+                        if (errorsFound) { // is this really needed here, program fatals out later on when errorsFound = true
```

Comment:
Maybe not, but it may help avoid problems later on by fataling here instead of later.  In any case, if it was going to fatal anyway, the end of this block seems reasonable.